src/chrome/content/rules/basekit.com.xml

```diff
@@ -1,79 +1,32 @@
 <!--
-	CDN buckets:
-
-		- basekit-tdk.s3.amazonaws.com
-		- basekit-image.s3.amazonaws.com
-		- d282ykz6vx01th.cloudfront.net
-		- d2f0ora2gkri0g.cloudfront.net
-		- d35onr1h4eb0bw.cloudfront.net
-
-
-	Nonfunctional hosts in *basekit.com:
-
-		- apidocs ʳ
-		- knowledge ᵈ
-
-	ᵈ Dropped
-	ʳ Refused
-
-
-	Problematic hosts in *basekit.com:
-
-		- custom-css ᵐ
-		- docs ᵐ
-		- resources ᵐ
-		- resellers *
-		- try *
-
-	* Protocol-relative inclusion of custom-css.basekit.com
-	ᵐ Mismatched
-
-
-	Insecure cookies are set for these hosts: ᶜ
-
-		- odense.basekit.com
-
-	ᶜ See https://owasp.org/index.php/SecureFlag
-
+	Invalid certificate:
+		basekit.com
+		custom-css.basekit.com
+		resources.basekit.com
 
 	Mixed content:
+		www.basekit.com
+		odense.basekit.com
 
-		- css on odense from fonts.googleapis.com ˢ
-
-		- Images, on:
-
-			- odense from d2f0ora2gkri0g.cloudfront.net ˢ
-			- odense from d35onr1h4eb0bw.cloudfront.net ˢ
-
-		- Bug on www from chrs-mln-krs.com ˢ
-
-	ˢ Secured by us, see https://www.paulirish.com/2010/the-protocol-relative-url/
+	Protocol-relative inclusion of custom-css.basekit.com:
+		resellers.basekit.com
+		try.basekit.com
 
 -->
 <ruleset name="BaseKit.com (partial)">
 
-	<!--	Direct rewrites:
-				-->
-	<target host="basekit.com" />
 	<target host="developers.basekit.com" />
-	<target host="odense.basekit.com" />
+	<target host="custom-css.basekit.com" />
 	<target host="resellers.basekit.com" />
 	<target host="scottishculture.basekit.com" />
-	<target host="www.basekit.com" />
-
-	<!--	Complications:
-				-->
-	<target host="custom-css.basekit.com" />
-	<target host="resources.basekit.com" />
-
+	<target host="try.basekit.com" />
```

Comment:
As far as I understand this host is a website builder. It's unlikely that there are major differences between all the sites or customers are allowed to insert active content. In this example only Google Fonts are affected.
Let's add the wildcard.